tests/unit/amo/components/TestSearchFilters.js

```diff
@@ -483,6 +523,45 @@ describe(__filename, () => {
         }),
       });
     });
+
+    it('does not pass sort=random when the featured checkbox is unchecked', () => {
```

Comment:
We plan on removing all code related to `featured`, but I guess until we do that we need this test for now.